viahtml/blocklist.py

```diff
@@ -0,0 +1,116 @@
+"""The abstract blocklist object."""
+
+import os
+import re
+from enum import Enum
+from logging import getLogger
+from urllib.parse import urlparse
+
+
+class Blocklist:
+    """A blocklist which URLs can be checked against.
+
+    The blocklist is stored at:
+      * https://hypothesis-via.s3-us-west-1.amazonaws.com/via-blocklist.txt
+
+    And is downloaded locally, on a loop by a cron job.
```

Comment:
'tis not a cron job

Reply:
Spot the comment written before finding out how it was actually done